### PR TITLE
feat(spark): Add an example to generate ldbc sample data from csv to graphar

### PR DESCRIPTION
**Pull request by acezen**
### Reason for this PR

as #486 describe

### What changes are included in this PR?

Add an example to generate ldbc sample data from csv to graphar

### Are these changes tested?

yes, have added the example to CI/CD

### Are there any user-facing changes?

no

### CODE REVIEW

File: maven-projects/spark/graphar/src/main/scala/org/apache/graphar/example/LdbcSample2GraphAr.scala [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+package org.apache.graphar.example
+
+import org.apache.graphar.graph.GraphWriter
+
+import org.apache.spark.sql.SparkSession
+
+object LdbcSample2GraphAr {
+
+  def main(args: Array[String]): Unit = {
+    // connect to the Neo4j instance
```
Comment (lixueclaire) [inline maven-projects/spark/graphar/src/main/scala/org/apache/graphar/example/LdbcSample2GraphAr.scala:29]: It seems that "Neo4j" is not correct here.

File: maven-projects/spark/graphar/src/main/scala/org/apache/graphar/example/LdbcSample2GraphAr.scala [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+      outputPath,
+      spark,
+      "LdbcSample",
+      vertexChunkSize,
+      edgeChunkSize,
+      fileType
+    )
+  }
+
+  // read data from Neo4j and put into writer
```
Comment (lixueclaire) [inline maven-projects/spark/graphar/src/main/scala/org/apache/graphar/example/LdbcSample2GraphAr.scala:71]: ditto

File: maven-projects/spark/graphar/src/main/scala/org/apache/graphar/example/LdbcSample2GraphAr.scala [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.graphar.example
```
Comment (lixueclaire) [inline maven-projects/spark/graphar/src/main/scala/org/apache/graphar/example/LdbcSample2GraphAr.scala:20]: Hi, @acezen . Could you please add a brief description for this example in the documentation or this [ReadMe page](https://github.com/apache/incubator-graphar/blob/main/maven-projects/spark/README.md)?

File: maven-projects/spark/graphar/src/main/scala/org/apache/graphar/example/LdbcSample2GraphAr.scala (hunk 1 above)
Comment (acezen) [reply maven-projects/spark/graphar/src/main/scala/org/apache/graphar/example/LdbcSample2GraphAr.scala]: fixed, thanks for pointing out.

File: maven-projects/spark/graphar/src/main/scala/org/apache/graphar/example/LdbcSample2GraphAr.scala (hunk 3 above)
Comment (acezen) [reply maven-projects/spark/graphar/src/main/scala/org/apache/graphar/example/LdbcSample2GraphAr.scala:20]: updated

**Review by lixueclaire — APPROVED** [review]
LGTM

**Pull request merged**